bundles/ranvier-commands/commands/look.js

```diff
@@ -50,12 +52,12 @@ module.exports = (srcPath) => {
   function lookRoom(state, player) {
     const room = player.room;
 
-    const [ line1, line2, line3 ] = getDirectionsMap(player);
+    const [ line1, line2, line3 ] = getCompass(player);
 
     // Render the room
-    Broadcast.sayAt(player, Broadcast.pad(new Array(90).join(' '), `<yellow><bold>${room.title}</bold></yellow>`) + line1);
-    Broadcast.sayAt(player, Broadcast.pad(new Array(60).join(' '), '--------------------------------------------') + line2);
-    Broadcast.sayAt(player, Broadcast.pad(new Array(60).join(' '), ' ') + line3);
+    Broadcast.sayAt(player, `<yellow><bold>${room.title}</bold></yellow>` + leftPad(line1, 164 - room.title.length));
```

Comment:
If we were doing fancy-pants telnet negotiation, this might be a good place to know the maximum width supported by a client. Is that even a thing. I don't know telnet.